src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,29 @@
+$starsWidth: 96px;
+$notActiveStar: url(../images/star.svg);
+$activeStar: url(../images/star-active.svg);
+
+.stars--content {
+  width: $starsWidth;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Are you sure you need `.stars--content` modifier? Why not use the `.stars` class?